client/components/UserDecalPanel.js

```diff
@@ -116,18 +144,16 @@ class UserDetailsPanel extends Component {
       redeeming,
     } = this.state
 
+    const loadingText = checkingEligibility ? 'Checking decal eligibility...' : 'Redeeming decal codes...'
+
     return (
-      <ReactTable
-        className="panel user-decals"
-        columns={this.columns}
-        data={decals}
-        defaultPageSize={10}
-        loading={checkingEligibility || redeeming}
-        loadingText={checkingEligibility ? 'Checking decal eligibility...' : 'Redeeming decal codes...'}
-        manual
-        minRows={5}
-        noDataText={this._renderNoDataText()}
-        showPagination={false} />
+      <div className="panel user-decals">
+        <header>Decal</header>
+        <div className="panel-content">
+          {(checkingEligibility || redeeming) ? (<div className="loading">{loadingText}</div>) : (this.renderDecalCodes(decals))}
+          {(decals.length) ? ('') : (this.renderNoDataText())}
```

Comment:
I would personally write this like:
```suggestion
          {!decals.length && this.renderNoDataText()}
```

React won't render anything if a block evaluates to `false`, `null`, or `undefined`.